src/fklearn/training/imputation.py

```diff
@@ -32,23 +33,41 @@ def imputer(df: pd.DataFrame,
         - If "mean", then replace missing values using the mean along the axis.
         - If "median", then replace missing values using the median along the axis.
         - If "most_frequent", then replace missing using the most frequent value along the axis.
+
+    fill_value : Any, (default=None)
+        if not None, use this as default value when some feature only contains NA values.
```

Comment:
im not sure this is clear; is the expected behavior for `fill_value` to be filled *strictly only when 100%* of the feature values is `na`? what if there's only 1 not missing (or any number > 0, fwiw)? when should i use `fill_value`? i think the description can be improved here.

Reply:
I'm trying (and failing) to mimic the `fill_value` description in https://scikit-learn.org/stable/modules/generated/sklearn.impute.SimpleImputer.html